src/core/utils/helpers/general.ts

```diff
@@ -1,12 +1,13 @@
 import { useEffect, useRef } from "react";
-import { CoverProps } from "../../components/cover/cover";
-import configuration, { getConf } from "../configuration";
-import { UseTextFunction } from "./text";
+import { CoverProps } from "../../../components/cover/cover";
+import configuration, { getConf } from "../../configuration";
 import {
-  WorkSmallFragment,
-  ManifestationsSimpleFragment
-} from "../dbc-gateway/generated/graphql";
-import { FaustId, Pid } from "./types/ids";
+  ManifestationsSimpleFragment,
+  WorkSmallFragment
+} from "../../dbc-gateway/generated/graphql";
+import { UseTextFunction } from "../text";
+import { FaustId, Pid } from "../types/ids";
+import { getUrlQueryParam } from "./url";
 
 export const orderManifestationsByYear = (
```

Comment:
I appreciate the introduction of `date` and `url` helpers. Have you considered adding a `work` or `material` helper?

Reply:
It depends if there is a direct link between those entities and the helper functions.
The core helpers are meant to be used on all areas of the application and that is why they are called generic names like url and date.
More specific helpers are also being used next to the various components.